pulsar-client/src/main/java/org/apache/pulsar/client/impl/ConnectionHandler.java

```diff
@@ -42,8 +43,9 @@ public class ConnectionHandler {
     private volatile ClientCnx clientCnx = null;
     @Getter
     @Setter
+    @VisibleForTesting
     // Since the `clientCnx` variable will be set to null at some times, it is necessary to save this value here.
-    private volatile int maxMessageSize = Commands.DEFAULT_MAX_MESSAGE_SIZE;
+    volatile int maxMessageSize = Commands.DEFAULT_MAX_MESSAGE_SIZE;
```

Comment:
There is already a getter and setter, we don't need to remove the `private` modifier here.

Reply:
Solved